centos/centos-8.x/centos-8.1-hpc/README.md

```diff
@@ -0,0 +1,24 @@
+# CentOS 8.1 HPC Image
+
+The CentOS 8.1 HPC Image includes optimizations and recommended configurations to deliver optimal performance,
+consistency, and reliability. This image consists of the following HPC tools and libraries:
+
+- Mellanox OFED
+- Pre-configured IPoIB (IP-over-InfiniBand)
+- Popular InfiniBand based MPI Libraries
+  - HPC-X
+  - IntelMPI
+  - MVAPICH2
+  - OpenMPI
+- Communication Runtimes
+  - Libfabric
+  - OpenUCX
+- Optimized librares
+  - AMD Blis
+  - AMD FFTW
+  - AMD Flame
+  - Intel MKL
+
+The MPI libraries are configured as environment modules. Users can select any preferred MPI flavor as follows:
```

Comment:
With your previous PR, this is not limited to just MPIs. Can you please update it accordingly?